docs/CONTRIBUTORS.rst

```diff
@@ -105,96 +34,73 @@ To work on the TUF project, it's best to perform a development install.
     $ python3 -m pip install -r requirements-dev.txt
 
 
-Auto-formatting
-===============
-
-CI/CD will check that new TUF code is formatted with `black
-<https://black.readthedocs.io/>`__ and `isort <https://pycqa.github.io/isort>`__.
-Auto-formatting can be done on the command line:
-::
-
-    $ black <filename>
-    $ isort <filename>
-
-or via source code editor plugin
-[`black <https://black.readthedocs.io/en/stable/editor_integration.html>`__,
-`isort <https://github.com/pycqa/isort/wiki/isort-Plugins>`__] or
-`pre-commit <https://pre-commit.com/>`__-powered git hooks
-[`black <https://black.readthedocs.io/en/stable/version_control_integration.html>`__,
-`isort <https://pycqa.github.io/isort/docs/configuration/pre-commit/>`__].
-
-
 Testing
 =======
 
-The Update Framework's unit test suite can be executed by invoking the test
-aggregation script inside the *tests* subdirectory. ``tuf`` and its
-dependencies must already be installed (see above).
-::
+With `tox <https://testrun.org/tox/>`_, the test suite can be executed in a
+separate *venv*. While a supported Python version must already be available.
 
-    $ cd tests
-    $ python3 aggregate_tests.py
+``tox`` will install ``tuf`` and dependencies, run the tests, linting, and
+coverage.
 
-Individual tests can also be executed. Optional '-v' flags can be added to
-increase log level up to DEBUG ('-vvvv').
 ::
 
-    $ python3 test_updater_ng.py # run a specific test file
-    $ python3 test_updater_ng.py TestUpdater.test_refresh_and_download # run a specific test
-    $ python3 test_updater_ng.py -vvvv TestUpdater.test_refresh_and_download # run test with DEBUG log level
+    $ tox
 
+Below, you will see more details about each steps managed by ``tox``, in case
+you need debug/run outside ``tox``.
 
-All of the log levels and the corresponding options that could be used for testing are:
+tests
```

Comment:
```suggestion
Unit Tests
```

Reply:
I fixed it, but IMO, Tests would be better once we don't have only unit tests running.